_books/ion-1-1/src/whats_new.md

```diff
@@ -256,53 +120,19 @@ field all together). In the following examples, let us define `(:make_struct c 5
-* **An exported macro table**, a list of macro definitions used to compactly encode complete values or partially populated containers.
-* **An unexported nested modules map**, a set of unique module names and their associated module definitions.
-
-While Ion 1.0 does not have modules, it is reasonable to think of Ion 1.0's local symbol table as a module that only has symbols,
-and whose macro table and nested modules map are permanently empty.
-
-Modules can be imported from the catalog (they subsume shared symbol tables) or defined locally.
-
-### Directives
-
-_Directives_ modify the encoding context.
-Syntactically, a directive is a top-level s-expression annotated with `$ion`.
-Its first child value is an operation name.
-The operation determines what changes will be made to the encoding context and which clauses may legally follow.
-
-```ion
-$ion::
-(operation_name
-    (clause_1 /*...*/)
-    (clause_2 /*...*/)
-    /*...*/
-    (clause_N /*...*/))
-```
 
-In Ion v1.1, there are three supported directive operations:
-1. [`module`](modules/directives.md#module-directives)
-2. [`import`](modules/directives.md#import-directives)
-3. [`encoding`](modules/directives.md#encoding-directives)
 
 ### Macro definitions
 
-Macros can be defined by a user either directly in a local module within an encoding directive or in a module
+Macros can be defined by a user either directly in a default module within an encoding directive or in a module
 defined externally (i.e., shared module). A macro has a name which must be unique in a module *or* it may have no name.
 
 Ion 1.1 defines a list of _system macros_ that are built-in in the module named `$ion`. Unlike the system symbol table,
 which is always installed and accessible in the local symbol table, the system macros are both always accessible to
 E-expressions and not installed in the local macro table by default (unlike the local symbol table).
 
-In Ion binary, macros are always addressed in E-expressions by the offset in the local macro table. System macros may
-be addressed by the system macro identifier using a specific encoding op-code. In Ion text, macros may be addressed by
-the offset in the local macro table (mirroring binary), its name if its name is unambiguous within the local encoding
-context, or by qualifying the macro name/offset with the module name in the encoding context. An E-expression can
+In Ion binary, macros are always addressed in E-expressions by integer macro address. For user macros this is the offset in the local macro table. System macros may be addressed by the system macro address using a specific encoding op-code. In Ion text, macros may be addressed by
+the offset in the local macro table (mirroring binary), by name if that name is unambiguous within the local encoding context, or by qualifying the macro name/offset with the module name in the encoding context. An E-expression can
```

Comment:
> by name if that name is unambiguous within the local encoding context

This is outdated. We now mandate that names are unambiguous. Macro names that are unqualified are implicitly referring to the default module, `_`.